ale_linters/python/pylint.vim

```diff
@@ -0,0 +1,9 @@
+" Author: keith <k@keith.so>
+" Description: pylint for python files
+
+call ale#linter#Define('python', {
+\   'name': 'pylint',
+\   'executable': 'pylint',
+\   'command': 'pylint --output-format text --msg-template="{path}:{line}:{column}: {msg_id} {msg}" --reports n %s',
```

Comment:
Ah ok, I was wondering why many linters were using stdin. I will update this. I don't _think_ we should need the wrapper here